trusted-committer/outline.md

```diff
@@ -0,0 +1,86 @@
+# Trusted Committer Learning Path Outline
+
+## Learning Goals
+
+- Definition of TC (why that name)
+- Understanding of the responsibilities of the Trusted Committer (TC)
+- Different ways to become TC
+- TC lifecycle (bringing on successors, sunsetting)
+- Useful skills/attitude/emotional maturity
+- TC antipatterns
+- Why is the TC role important for IS success
+
+## Target Audience
+
+- Current TCs
+- People who work/interface with TCs 
+    - Contributors
+    - Potential future TCs
+    - Managers of TCs
+
+## Outline
+
+### Definition
+
+- Glossary
+    - Contributor/Guest
+        - Responsible for "fitting in"/conforming to the host team and it's
+          expectations/processes.
+    - Product Owner (distinction btw. Guest and Host PO)
+    - TC
+        - TC has rights to push code closer to production than a contributor
+        - also known as _Host_
+        - has privileges that an admin or Apache committer would have but it is
+          a job, irrespective of the permissions. 
+        - Maintainer/Owner (conflict with _maintainer_ role in GitHub)
+        - Committer is not an IS term. Apache uses this term to denote a tech
+          leadership role (TC++)
+        - The TC role is defined by its responsibilities, rather than by its
+          privileges.
+
+- at PayPal, the discussion surrounding the name of the role was very
+  challenging.  i. e.: the name matters.
+
+### Responsibilities
+
+**Tech oriented**
+- Keep SW development running smoothly
+- Ensure quality and longevity of the SW at a strategic level
+- Ensure that decisions regarding quality and longevity of the SW are
+  enact/follow/implement/...
+- Own decisions regarding the community/process/technology, even if they didn't
+  make it themselves.
+- Reviewing and accepting contributions (including making sure they conform to
+  the guidelines)
+- Reponsible for communicating contribution guidelines (possibly also
+  validating and setting guidelines)
+- Responsible for making sure undocumented requirements will be documented 
+  (not necessarily do it herself)
+
+**Community oriented**
+- TC has to be aware of the communities needs
+- TC has to advocate the communitities interests
+- Reducing the barriers for entry into the community
+- Onboarding new contributors
+    - Making sure the new contributor is prepared/educated to make the next
+      contribution
+- Upleveling the capabilities of the community members to use and contribute to
+  the project  
+    - Mentoring/Coaching during the process of contributors making a
+      contribution
+- Set up the right process for the community 
+- Fostering the community
+    - enforcing the code of conduct
+- 
+
+### Lifecycle
+
+- There might well be multiple TCs in one project
+- 
+
+## Ideas
+
+- Possibly two videos: one for being and one for becoming a TC
```

Comment:
Like this idea.

Reply:
Actually, that's a nice idea. Did we do any of that (still need to go through all the Screenlight things)? Would we want to have small parts of it included in the contributor segment (#37) along the lines of "from contributor to trusted committer" - that's something that would likely work with open source projects, I'm not sure how well it works for all inner source projects, with the grass-roots inner source projects that should probably be fine.